lib/screens/util.ex

```diff
@@ -179,4 +181,22 @@ defmodule Screens.Util do
   def to_set(id) when is_binary(id), do: MapSet.new([id])
   def to_set(ids) when is_list(ids), do: MapSet.new(ids)
   def to_set(%MapSet{} = already_a_set), do: already_a_set
+
+  def get_color_for_route(route_id, route_type \\ nil)
```

Comment:
Good idea to move this since it doesn't quite belong in RoutePill. Maybe Screens.Routes.Route would be even better than Screens.Util, though?

Reply:
Good idea, Route does make a bit more sense.